portfolio/src/main/webapp/index.html

```diff
@@ -7,31 +7,71 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <!-- Sticky Navbar -->
```

Comment:
Happy to see these comments here! It definitely helps you keep the context in mind when scrolling through files written in relatively verbose languages like HTML.

Reply:
Thanks!